activestorage/CHANGELOG.md

```diff
@@ -1,11 +1,11 @@
 *   Don't stream responses in redirect mode
 
     Previously, both redirect mode and proxy mode streamed their
-	responses which caused a new thread to be created, and could end
-	up leaking connections in the connection pool. But since redirect
-	mode doesn't actually send any data, it doesn't need to be
-	streamed.
+		responses which caused a new thread to be created, and could end
+		up leaking connections in the connection pool. But since redirect
+		mode doesn't actually send any data, it doesn't need to be
+		streamed.
```

Comment:
Hey good catch.
I think it should be just 4 spaces though.
See: https://raw.githubusercontent.com/rails/rails/7-0-stable/activestorage/CHANGELOG.md
```suggestion
    responses which caused a new thread to be created, and could end
    up leaking connections in the connection pool. But since redirect
    mode doesn't actually send any data, it doesn't need to be
    streamed.
```